app/templates/client/app/main/main.controller(coffee).coffee

```diff
@@ -1,22 +1,28 @@
 'use strict'
 
-angular.module '<%= scriptAppName %>'
-.controller 'MainCtrl', ($scope, $http<% if (filters.socketio) { %>, socket<% } %>) ->
-  $scope.awesomeThings = []
-
+MainController = ($scope, $http<% if (filters.socketio) { %>, socket<% } %>) ->
+  self = this
+  @awesomeThings = []
   $http.get('/api/things').then (response) ->
```

Comment:
With no server options this is the last statement in the controller and coffee returns the last statement in a function. Also, when using `controller as` syntax if the controller returns a value then it is bound to the scope as the controller. So `this` should be the last statement.